### PR TITLE
✨  Update copyright headers

### PR DESCRIPTION
**Pull request by singhsaurabh**
#### What kind of change does this PR introduce?

_It is copyright notice format update._

- [x] PR title follows the guidelines defined in our [pull request documentation](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process)

#### What is the current behavior?

_Current copyright notice show project name. And also not showing SPDX header._
 
#### What is the new behavior (if this is a feature change)?**

_New copyright notice changed to ORG name  `OpenSSF`. Added option `-s` to `addlicense`  to generate `SPDX` header also._ 

- [ ] Tests for the changes have been added (for bug fixes/features)

#### Which issue(s) this PR fixes

https://github.com/ossf/scorecard/issues/1534

#### Does this PR introduce a user-facing change?

For user-facing changes, please add a concise, human-readable release note to
the `release-note`

(In particular, describe what changes users might need to make in their
application as a result of this pull request.)

```release-note

```

### CODE REVIEW

**Review by justaugustus — APPROVED** [review]
Thanks for this, @singhsaurabh!

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -680,5 +680,5 @@ jobs:
         run: |
           go env -w GOFLAGS=-mod=mod
           go install github.com/google/addlicense@2fe3ee94479d08be985a84861de4e6b06a1c7208
-          addlicense -ignore "**/script-empty.sh" -ignore "pkg/testdata/**" -ignore "checks/testdata/**" -l apache -c 'Security Scorecard Authors' -v *
+          addlicense -s -ignore "**/script-empty.sh" -ignore "pkg/testdata/**" -ignore "checks/testdata/**" -l apache -c 'OpenSSF Authors' -v *
```
Comment (azeemshaikh38) [inline .github/workflows/main.yml:683]: I thought we decided this should continue to be `Scorecard Authors` but without the year/date? @justaugustus

Comment (justaugustus) [reply .github/workflows/main.yml:683]: > I thought we decided this should continue to be `Scorecard Authors` but without the year/date? @justaugustus

From https://github.com/ossf/scorecard/issues/1534#issuecomment-1048102540:

> So I guess to more concretely answer this, a few things need to happen:
> 
>     1. Scorecard maintainers (@naveensrinivasan @azeemshaikh38 @laurentsimon me) need to decide if it's fine to elide the copyright date in all files (I'm personally fine with it)
> 
>     2. OpenSSF and scorecard maintainers need to decide if it's preferred to use `OpenSSF Authors` as the copyright holder for content in this repo and potentially elsewhere
> 
>     3. Someone (@singhsaurabh) will update the copyright headers in this repo (as well as any tests that enforce them) based on the decisions on 1 and 2

I thought we landed on 2: `OpenSSF Authors` (no date) or rather, haven't heard anything to the contrary.

Comment (azeemshaikh38) [reply .github/workflows/main.yml:683]: > I thought we landed on 2: `OpenSSF Authors` (no date) or rather, haven't heard anything to the contrary.

In the last Scorecard bi-weekly we decided on `1` (see https://github.com/ossf/scorecard/discussions/1684)? `2` would be ideal, but we need confirmation from LF for this.

**Review by laurentsimon — APPROVED** [review]
Thanks!

File: .github/workflows/main.yml (hunk 1 above)
Comment (laurentsimon) [inline .github/workflows/main.yml:683]: will this break subsequent PRs unless we also update each file's headers in this PR?

Comment (naveensrinivasan) [reply .github/workflows/main.yml:683]: I think it will

**Comment by naveensrinivasan** [comment]
Can this PR also change all the headers with the new format?

File: .github/workflows/main.yml (hunk 1 above)
Comment (justaugustus) [reply .github/workflows/main.yml:683]: > will this break subsequent PRs unless we also update each file's headers in this PR?

From https://github.com/google/addlicense:

> It modifies all source files in place and avoids adding a license header to any file that already has one.

So no, it shouldn't cause failures in existing files, but for PRs that add new files requiring a header, they will error on the license check.

Comment (naveensrinivasan) [reply .github/workflows/main.yml:683]: Cool, Thanks

**Comment by naveensrinivasan** [comment]
> Can this PR also change all the headers with the new format?

This is not a blocker. It can be a separate PR.

**Comment by justaugustus** [comment]
> > Can this PR also change all the headers with the new format?
> 
> This is not a blocker. It can be a separate PR.

Agreed. We should lock enforcement of the new header in place first.

That said, we need to sync with LF to confirm the new header is fine to use, so let's hold this PR in the meantime.

**Comment by singhsaurabh** [comment]
Thank You All for review the PR. 
Regards,

**Pull request closed without merging**